docs/metrics.md

```diff
@@ -145,6 +145,19 @@ and the transformation to generate the exported metric.
 |||  
 
 
+### flows length histogram
+| **Description** | A histogram of flowlog bytes | 
+|:---|:---|
+| **Details** | Flows length distribution over time | 
+| **Usage** | Evaluate flows length behavior including mice/elephant use-case | 
```

Comment:
is the mention of "mice/elephants" here (and below) still intended?

Reply:
Eran has updated it recently so I guess it is intended 
https://github.com/netobserv/flowlogs-pipeline/pull/164#discussion_r841189736

Reply:
@KalmanMeth this is the only place where we explain that the histogram is useful for the mice/elephant use-case --- hope that this make sense to you